skimage/color/colorconv.py

```diff
@@ -1455,3 +1457,37 @@ def _prepare_lab_array(arr):
     if shape[-1] < 3:
         raise ValueError('Input array has less than 3 color channels')
     return dtype.img_as_float(arr, force_copy=True)
+
+
+def img_to_cmap(img, cmap):
+    """Draw a grayscale image by using the given colormap.
+
+    Parameters
+    ----------
+    img : (M, N) array
+        The input image.
+    cmap : string or :class:`matplotlib.colors.Colormap` instance.
+        The colormap to use
+
+    Returns
+    -------
+    out: (M, N, 3) array
+        The image drawn with the specified colormap in RGB format.
+
+    Examples
+    --------
+    >>> from skimage import data, color
+    >>> out = color.img_to_cmap(data.camera(), 'afmhot')
+    """
+
+    cmap = plt.get_cmap(cmap)
+    img = img_as_ubyte(img)
```

Comment:
Is the `img_as_ubyte` conversion really necessary? It would be good for this function to work for non-standard float images.